src/components/benefits/styledBenefits.js

```diff
@@ -0,0 +1,62 @@
+import styled from "styled-components/macro";
+
+export const BenefitsSection = styled.div`
+  height: 751px;       /* 871px (slide to next section) - 120px (padding) */
+  margin: 0 auto;
+  position: relative;
+
+  @media( max-width: 1280px) {
+  }
+  @media( max-width: 1023px) {
+    height: 878px;
+  }
+  @media(max-width: 767px) {
+    height: auto;
+    padding: 118px 16px 0;
+    display: flex;
+    flex-direction: column;
+    justify-contents: center;
+    align-items: center;
+    gap: 75px;
+  }
+`;
+
+export const BenefitGroup = styled.div`
+  display: flex;
+  gap: 40px;
```

Comment:
The first image/text group has a smaller gap (25px) at higher screen widths, but I wouldn't mind if you didn't change this.  Seems like poor design to make them all different

Reply:
Yeah I've started to get to the point where if there's a small difference like this, I may deviate from the design a bit for convenience